code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -81,6 +142,7 @@
 		. += "<span class='notice'>The status display reads: Efficiency at <b>[efficiency*100]%</b>.</span>"
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/Destroy()
+	QDEL_NULL(occupant_vis)
```

Comment:
```suggestion
	if(occupant_vis)
		vis_contents -= occupant_vis
		QDEL_NULL(occupant_vis)
```
Let's remove references to avoid hard deletions.

Reply:
I'll change it so that occupant_vis does not store cryo as well, rather taking it as a parameter for the icon update.